EIPS/eip-7873.md

```diff
@@ -175,7 +175,7 @@ This change poses no risk to backwards compatibility, as it is introduced at the
 
 The transactions of the new type are invalid until this change activates.
 
-Contract creation options do not change for legacy bytecode.
+Contract creation options do not change for legacy bytecode, including how existing transactions behave when encountering code that may look like an EOF container.
```

Comment:
```suggestion
Contract creation options do not change for legacy bytecode, including how existing transactions with `to: nil` behave when encountering code in their calldata that may look like an EOF container.
```

Otherwise
> encountering code that may look like an EOF container.

is ambiguous.